style.css

```
@@ -0,0 +1,55 @@
+body {
+  background-image: url("./images/bg.jpeg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: "Varela Round", sans-serif;
+}
+
+h1, h2, h3 {
+  font-family: "Pacifico", cursive;
+  text-align: center;
+  font-size: 72px;
+  color: orange;
+}
+
+h2 {
+  font-size: 60px;
+}
+
+h3 {
+  font-size: 40px;
+}
+div{
+  text-align: center;
+}
+
+button {
+  color: orange;
+  background-color: #ffffff;
+  background-color: rgba( 255, 255, 255, 0 );
+  padding: 15px 32px;
+  text-align: center;
+  font-family: "Pacifico", cursive;
+  text-decoration: none;
+  font-size: 50px;
+  margin: 4px 2px;
+  cursor: pointer;
+  width: 300px;
+  height: 150px;
+  cursor: "pointer"
+}
+
+input {
+  width: 300px;
+  height: 100px;
+  font-size:25px;
+  text-align:center;
```

속성 앞뒤의 space도 통일해서 입력해주세요.
```suggestion
  font-size: 25px;
  text-align: center;
```